src/components/Header.js

```diff
@@ -57,14 +103,75 @@ const Header = ({ menuContainer }) => {
         />
         <Typography className={classes.title}>Your Story Matters</Typography>
       </Box>
-      {/* TODO: Will actually be breathing timer */}
       <Box p={2}>
-        <IconButton onClick={() => { setDrawerOpen(true); }}>
-          <NaturePeople />
+        <IconButton onClick={() => { setModalOpen(true); }}>
+          <SvgIcon component={BreathIcon} />
         </IconButton>
 
       </Box>
 
+      <Dialog
```

Comment:
Just for future maintenance… could you move this to a separate component that's self contained?

Reply:
Done